scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-6x.yaml

```diff
@@ -60,6 +60,64 @@ QFN-64-1EP_8x8mm_P0.4mm_EP6.5x6.5mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-64-1EP_9x9mm_P0.5mm_EP4.35x4.35mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.ftdichip.com/Support/Documents/DataSheets/ICs/DS_FT2232H.pdf#page=57'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  body_size_x:
+    nominal: 9
+    tolerance: 0
+  body_size_y:
+    nominal: 9
+    tolerance: 0
+  overall_height:
+    minimum: 0.8
+    nominal: 0.85
+    maximum: 0.9
+
+  lead_width:
+    minimum: 0.18
+    nominal: 0.25
+    maximum: 0.30
+  lead_len:
+    minimum: 0.35
+    nominal: 0.4
+    maximum: 0.45
+
+  EP_size_x:
+    minimum: 4.3
+    nominal: 4.35
+    maximum: 4.4
+  EP_size_y:
+    minimum: 4.3
+    nominal: 4.35
+    maximum: 4.4
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [4, 4]
```

Comment:
Make this `[3, 3]` to go nicely with the 4x4 via grid.

Reply:
Done